keepassxc-browser/popups/popup.html

```diff
@@ -95,5 +89,12 @@
         </div>
       </div>
     </div>
+
+    <script type="text/javascript" src="../browser-polyfill.min.js"></script>
+    <script type="text/javascript" src="../global.js"></script>
+    <script type="text/javascript" src="../jquery-3.3.1.min.js"></script>
+    <script type="text/javascript" src="../options/bootstrap.min.js" /></script>
+    <script type="text/javascript" src="popup_functions.js" /></script>
+    <script type="text/javascript" src="popup.js" /></script>
```

Comment:
For that, you can also easily use the [defer attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script) instead of moving stuff around. 😄